data/permissions.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @title Permission Management
+ * @difficulty beginner
+ * @tags cli
+ * @run <url>
+ * @resource {https://deno.land/api@v1.29.2?s=Deno.Permissions} Doc: Deno.Permissions
```

Comment:
How about linking to `https://deno.land/api?s=Deno.Permissions` instead? That automatically navigates to the latest version of the doc, and it might be less confusing to the users.

Reply:
SGTM, I had assumed that the policy was pining everything to a version (especially since the permissions api may actually get expanded with sync versions and all). Is this a policy for all denobyexample articles or just for stable apis?

Now that I think about it I don't think there are examples for non-stable apis (std(?)) so maybe I'm just thinking about this wrong.